cluster/test-deploy/openshift-ansible-aws/vars-origin.yaml

```diff
@@ -0,0 +1,119 @@
+ansible_become: true
+ansible_become_sudo: true
+
+openshift_deployment_type: origin
+openshift_additional_repos:
+- id: origin
+  name: Origin
+  baseurl: "{{ lookup('env','RPM_REPO_BASEURL') }}"
```

Comment:
Why did you call this `RPM_REPO_BASEURL` instead of `RPM_REPO`, which is what ci-operator generates?

Reply:
`RPM_REPO` is a path to .repo file and openshift-ansible doesn't have a var for a full repo file yet (see https://github.com/openshift/openshift-ansible/issues/9859)